src/control/server/mgmt_pool.go

```diff
@@ -0,0 +1,340 @@
+//
+// (C) Copyright 2018-2020 Intel Corporation.
```

Comment:
If this is a new file, the copyright year should be 2020.

Reply:
done